flower.rb

```diff
@@ -12,6 +12,9 @@ class Flower < Formula
   depends_on "python3"
 
   def install
+    vendor_site_packages = libexec/"vendor/lib/python#{Language::Python.major_minor_version "python3"}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", vendor_site_packages
```

Comment:
Sorry for being so persistent, but I truly think that this specific change is not needed. There should be no need to modify `flower.rb` at all. It's a bug in the ARPC package; not the Flower package.

I just ran `vi $(git grep -l pypi.python)` on `homebrew-core` and took a look at how other packages do this. The idiomatic way of solving this is to do this after installation:

    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])

That way `aprotc` is always invoked with the right `${PYTHONPATH}`.

Reply:
I like this a lot better, yep. I'll take a look